doc/ReleaseNotes/_ReleaseNotes.md

```diff
@@ -148,6 +149,7 @@
  * 146648 [Android] fixed ListView grouped items corruption on scroll
  * [Wasm] Fix `ListView` recycling when the `XamlParent` is not available for `AutoSuggestBox`
  * 147405 Fix NRE on some MediaTransportControl controls
+ * #139 Update Uno.SourceGenerationTasks to improve build performance
```

Comment:
What is the required action for a project using uno to adapt to this breaking change?

Reply:
nevermind! thought this was under breaking changes !